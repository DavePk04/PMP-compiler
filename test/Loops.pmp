begin
  while a < b do
  begin
    a:=a+1...
  end...
end
