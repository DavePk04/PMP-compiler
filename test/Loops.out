token: begin	lexical unit: BEG
token: while	lexical unit: WHILE
token: a	lexical unit: VARNAME
token: <	lexical unit: SMALLER
token: b	lexical unit: VARNAME
token: do	lexical unit: DO
token: begin	lexical unit: BEG
token: a	lexical unit: VARNAME
token: :=	lexical unit: ASSIGN
token: a	lexical unit: VARNAME
token: +	lexical unit: PLUS
token: 1	lexical unit: NUMBER
token: ...	lexical unit: DOTS
token: end	lexical unit: END
token: ...	lexical unit: DOTS
token: end	lexical unit: END
